src/Compilers/CSharp/Portable/Declarations/SingleTypeDeclaration.cs

```diff
@@ -26,6 +26,7 @@ internal enum TypeDeclarationFlags : byte
             HasBaseDeclarations = 1 << 3,
             AnyMemberHasAttributes = 1 << 4,
             HasAnyNontypeMembers = 1 << 5,
+            HasConstraints = 1 << 6,
```

Comment:
The naming convention from previous entries is a bit confusing. Should this be `HasAnyConstraints` or `HasConstraints`? I'm fine either way.